### PR TITLE
Remove cur feature flag

### PR DESCRIPTION
**Pull request by brendasukh**
## Hey, I just made a Pull Request!
- Remove cost insights currencies feature flag, enabling users to see costs in terms of other currencies

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by timbonicus** [comment]
Can you attach a screenshot of how this looks now? I know we have a task to revisit the design before removing the feature flag.

**Comment by brendasukh** [comment]
@timbonicus yep! After some discussion w/ Janisa, we decided to release it as-is for feedback. We figured it would be better to at least get some eyes on the feature itself and gather feedback, rather than waiting for design or spending more time trying to figure out design, especially if it will be refactored based on feedback. 
Screen Shot 2020-11-16 at 2 17 22 PM

File: plugins/cost-insights/src/components/CurrencySelect/CurrencySelect.tsx [hunk 1]
```diff
@@ -57,6 +57,7 @@ export const CurrencySelect = ({
       onChange={handleOnChange}
       value={currency.kind || NULL_VALUE}
       renderValue={renderValue}
+      style={{ marginRight: '15px' }}
```
[the remark this answers is not in the document]
Comment (brendasukh) [reply plugins/cost-insights/src/components/CurrencySelect/CurrencySelect.tsx:60]: @ryanvazquez hmm, yeah I had added the styling directly to the dropdown after noticing the weird misalignment the `<Box/>` causes around the currency dropdown. I couldn't figure out a way around it or why it was happening 🤔 
Screen Shot 2020-11-16 at 2 26 16 PM

[the remark this answers is not in the document]
Comment (ryanvazquez) [reply plugins/cost-insights/src/components/CurrencySelect/CurrencySelect.tsx:60]: Screen Shot 2020-11-16 at 2 49 19 PM

Probably the `minHeight` prop on the child. Should be on the parent.


```ts
  const CostOverviewBanner = () => (
    <Box
      px={3}
      pt={6}
      display="flex"
      justifyContent="space-between"
      alignItems="center"
      minHeight={40}
    >
      <Box>
        <Typography variant="h4">Cost Overview</Typography>
        <Typography classes={classes}>
          Billing data as of {lastCompleteBillingDate}
        </Typography>
      </Box>
      <Box display="flex">
        <Box mr={1}>
          <CurrencySelect
            currency={currency}
            currencies={config.currencies}
            onSelect={setCurrency}
          />
        </Box>
        <ProjectSelect
          project={pageFilters.project}
          projects={projects || []}
          onSelect={onProjectSelect}
        />
      </Box>
    </Box>
  );
```

[the remark this answers is not in the document]
Comment (ryanvazquez) [reply plugins/cost-insights/src/components/CurrencySelect/CurrencySelect.tsx:60]: Can add `mx={-2}` if we want it to overflow horizontally.

```ts
    <Box
      px={3}
      pt={6}
      mx={-2}
      display="flex"
      justifyContent="space-between"
      alignItems="center"
      
```

Screen Shot 2020-11-16 at 2 54 59 PM

**Comment by timbonicus** [comment]
Can we wrap an input label around the select, similar to the `Compare to:` metric label? Something like `Convert to:` or `Currency:` or `Show change as:`. I feel like it's opaque what the dropdown is for right now.

**Review by timbonicus — APPROVED** [review]
🙌

**Pull request merged**